package.json

```diff
@@ -44,6 +44,7 @@
         "plotly.js": "^1.58.4",
         "popper": "^1.0.1",
         "react-bootstrap": "^1.6.1",
+        "react-tooltip": "^4.2.21",
```

Comment:
I believe I already have react-tooltips in here somewhere - see the saving filters feature on the flights page

Reply:
I installed a package, not realizing that there was already another way of creating a tooltip.